plugins/trendmicro_visionone/requirements.txt

```diff
@@ -0,0 +1,8 @@
+# List third-party dependencies here, separated by newlines. 
+# All dependencies must be version-pinned, eg. requests==1.2.0
+# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+requests==2.28.1
```

Comment:
Unless a specific version is required, requests is version pinned inside the insightconnect-plugin-runtime.